config.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	log "github.com/Sirupsen/logrus"
+	vc "github.com/containers/virtcontainers"
+	"github.com/containers/virtcontainers/pkg/oci"
+)
+
+const (
+	defaultRuntimeLib           = "/var/lib/cc-runtime"
+	defaultRuntimeRun           = "/var/run/cc-runtime"
+	defaultRuntimeConfiguration = "/etc/cc-runtime/configuration.toml"
```

Comment:
I was thinking that we should stick everything (runtime/proxy/shim?) config files under the same directories, say `clearcontainers` for 3.0:

```
/etc/clearcontainers/configuration.tom
/var/run/clearcontainers/proxy.sock
/var/run/clearcontainers/proxy-shim.sock
/var/run/clearcontainers/runtime/
```

(can be done at a later time if at all, of course)